osu.Server.Queues.ScoreStatisticsProcessor/Models/SoloScore.cs

```diff
@@ -54,7 +54,7 @@ public string data
         public double? pp { get; set; }
 
         public ulong? legacy_score_id { get; set; }
-        public uint? legacy_total_score { get; set; }
+        public uint legacy_total_score { get; set; }
```

Comment:
is this what we're going forward with just to fix tests? even though it is the osu-web image that isn't matching production? not even a todo in sight to get this fixed later?

Reply:
apparently it was an oversight and it's non-null on production and osu-web. my local test setup was fucked.

Reply:
> i edited my message showing actual production `CREATE TABLE` soon after posting if you want to re-check

well ok thanks for clarifying. saves me from wondering if i had a stroke earlier today